screensaver.fTVscreensaver/changelog.txt

```diff
@@ -0,0 +1,23 @@
+0.1.2
```

Comment:
Please try to use only the news element in the future

Reply:
Leftover file. Missed it. Will be removed.
There are reasons why I don't use the news element: Identation. It's a pita to format the text correctly. And a changelog.txt is more useful by looking at source code on GitHub or anywhere else. I wish Kodi never had removed the native changelog.txt support.